src/common.ts

```diff
@@ -105,18 +106,23 @@ export async function validateElectronApp(
   debug('Checking for a package.json file');
 
   const bundledPackageJSONPath = path.join(bundledAppDir, 'package.json');
-  if (!(await fs.pathExists(bundledPackageJSONPath))) {
+  if (!fs.existsSync(bundledPackageJSONPath)) {
     const originalPackageJSONPath = path.join(appDir, 'package.json');
     throw new Error(
       `Application manifest was not found. Make sure "${originalPackageJSONPath}" exists and does not get ignored by your ignore option`,
     );
   }
 
   debug('Checking for the main entry point file');
-  const packageJSON = await fs.readJson(bundledPackageJSONPath);
+  const { default: packageJSON } = await import(
+    pathToFileURL(bundledPackageJSONPath).toString(),
+    {
+      with: { type: 'json' },
+    }
+  );
```

Comment:
This is dangerous and a change in behavior, specifically it goes through the module cache meaning that changes on disk will not be re-read if this is re-imported